src/bin/console/console.cpp

```diff
@@ -1078,6 +1086,58 @@ bool Console::_handle_rollback() {
   return false;
 }
 
+bool Console::_verify_plan(const std::vector<std::shared_ptr<AbstractLQPNode>>& lqp) {
+  const auto& sm = StorageManager::get();
```

Comment:
please don't use abbreviations in newly written code. I know some old tests do this but newly written code shouldn't